.github/workflows/helm-chart-test.yml

```diff
@@ -13,105 +13,46 @@ permissions:
   contents: read
 
 jobs:
-  lint-test:
-    name: "Lint Tests with ct"
-    runs-on: ubuntu-latest
+  build-and-test:
+    # Skip job based on the commit message, only works in push to branches for now
+    if: contains(toJson(github.event.commits), '[skip ci]') == false
+    name: Build & test Docker images with Helm charts
+    runs-on: ubuntu-20.04
```

Comment:
Why `ubuntu-20.04`?

Reply:
I refer to the workflow of docker image build & test, so pick up this node OS.
Do I need to do a bulk update for other workflow to use the same node `ubuntu-latest`?

Reply:
> Why `ubuntu-20.04`?

I have done the bulk updates for all workflow to use the same `ubuntu-latest`